raw/src/requests/set_webhook.rs

```diff
@@ -0,0 +1,50 @@
+use requests::*;
```

Comment:
Can you add doc strings for `SetWebhook` methods? We definitely should clarify `max_connections` limits.

Reply:
What do you mean by adding doc strings? The fields in this method is exactly the same as telegram's official bot doc (including 'max_connections'). As I checked other methods, they all only have a single sentence or so describing what the method does. So does this one.